keylime/cloud_verifier_tornado.py

```diff
@@ -474,36 +633,50 @@ def main(argv=sys.argv):
 
     config = configparser.ConfigParser()
     config.read(common.CONFIG_FILE)
-
     cloudverifier_port = config.get('cloud_verifier', 'cloudverifier_port')
+    os.umask(0o077)
```

Comment:
Can we throw some comments in to this section to describe what's happening?

Reply:
agree, sure

Reply:
Since `init_dn` has been removed we need to create the directory path for where the sqlite database is stored. I think this is better situated under the sqlite conditional. You will see this in my next commit.